spec/50-privacy.md

```diff
@@ -1,21 +1,23 @@
 # Privacy
 
-Requirements to propagate headers to downstream services as well as storing values of these headers opens up potential privacy concerns. Trace vendors MUST NOT use `traceparent` and `tracestate` fields for any personally identifiable or otherwise sensitive information. The only purpose of these fields is to enable telemetry correlation.
+Requirements to propagate headers to downstream services as well as storing values of these headers opens up potential privacy concerns. Trace vendors MUST NOT use `traceparent` and `tracestate` fields for any personally identifiable or otherwise sensitive information. The only purpose of these fields is to enable trace correlation.
```

Comment:
`telemetry` is more generic than `trace` in a sense that logs can also be correlated with trace context. Why do you think `trace` is better? Maybe `event` can be used here

Reply:
We are using tracing throughout the document so we should be fine for now. We can still decide to adjust in a final language review.